cli/installer/install-azd.ps1

```diff
@@ -364,7 +359,18 @@ try {
                 Copy-Item "$tempFolder/decompress/$binFilename" $outputFilename  -ErrorAction Stop | Out-Null
             }
         } else {
-            Copy-Item "$tempFolder/decompress/$binFilename" $outputFilename  -ErrorAction Stop | Out-Null
+            Write-Verbose "Installing MSI" -Verbose:$Verbose
+            $MSIEXEC = "${env:SystemRoot}\System32\msiexec.exe"
+            $installProcess = Start-Process $MSIEXEC `
+                -ArgumentList @("/i", $releaseArtifactFilename, "/qn", "INSTALLDIR=`"$InstallFolder`"") `
```

Comment:
Just wondering - I know you mentioned we might be showing some level of UI to show the MIT License and some other notices.  Does that mean we should not be passing `/qn`?  Would this suppress that UI?

Reply:
/qn does suppress the MSI ui and consents to all prompts (we won't have any prompts in this case). The install script does write the telemetry notice to the console in the way that it always has. 

Since the powershell script is already non-interactive I didn't want to break the non-interactive behavior by launching an interactive install process.